languages/en.ini

```diff
@@ -714,6 +714,8 @@ Last Name = "Last Name"
 Last Search Result = "Last Search Result"
 less = "less"
 less_ellipsis = "less…"
+libguidesaz_recommendations = "Database Results"
```

Comment:
I should note that I found one other "databases" recommendation module, for Summon, where the default text is a vague 

> "You may find additional resources here:"

If that's a pattern we have to follow then fine.  But my text is more in line with the other SearchObject recommendation modules.

Reply:
This makes sense to me -- I like the pattern of having a module-specific translation string, since that allows targeted customization.